.gitignore

```diff
@@ -15,3 +15,4 @@
 /var/
 /.nightly_password
 /README.html
+site_administrator_emails.json
```

Comment:
It's not a good practice to write into the egg. 

Imagine we have a egg cache and 20 site are sharing this egg cache? Furthermore usually all the plone site have the same ID called `platform`, since they are in a mount-point. 

There are several approaches how to store and configure pass configuration into a package. 

You can assume a hard coded path, but this should be somewhere in the buildout directory. 
For example a dot-file under ${buildout-dir}/.linkchecker-site_administrator_emails.json
Also a buildout config would be a good option, since it's only possible to install with buildout, we usually put configs there.

Reply:
I'll address this in issue #29